client/question/domain/FeedbackObjectFactory.js

```diff
@@ -186,6 +186,19 @@ tie.factory('FeedbackObjectFactory', [
       }
     };
 
+
+    /**
+     * Returns all of the feedback provided as a JSON dict, preserving type
+     * and content information.
+     *
+     * @returns [{string, string}] The feedback paragraphs list of dicts.
```

Comment:
Should the second arg of the type be Object?

Reply:
Whoops, yup!